### ISSUE TITLE
Overlapping plots

### ISSUE THREAD

**Issue by ImmanuelSamuel** [bokeh/bokeh#5193]
The figure on the right and the table on the left overlap

```
from random import random
import numpy as np

from bokeh.layouts import column, row, widgetbox, layout
from bokeh.models import Button
from bokeh.palettes import RdYlBu3
from bokeh.plotting import figure, curdoc
from bokeh.models.widgets import MultiSelect, Dropdown, Select
from bokeh.models import ColumnDataSource
from bokeh.models.widgets import DataTable, DateFormatter, TableColumn

p1 = figure(plot_width=300, plot_height=300)
data = dict(x=[i for i in range(10)], y=[i for i in range(10)])
source3 = ColumnDataSource(data)
r1 = p1.circle('x', 'y', size=20, color="navy", alpha=0.5, source=source3)    

labels = np.array(np.arange(10), dtype=str).tolist()
menu = [('abc'+each1, 'abc'+each2) for each1, each2 in zip(labels,labels)]
dropdown = Dropdown(label="Dropdown button", menu=menu)

data = dict(downloads=['abc'+ str(i) for i in range(30)])
source = ColumnDataSource(data)
columns = [TableColumn(field="downloads", title="Downloads")]
data_table = DataTable(source=source, columns=columns, width=300, height=300,
                       selectable=True, row_headers=False)

l = row(widgetbox(dropdown, data_table), p1)
curdoc().add_root(l)
```

bokeh serve this to replicate

issue
![selection_021](https://cloud.githubusercontent.com/assets/7865018/18651809/b02d5282-7e9b-11e6-855a-813deadf3710.png)

**Comment by bryevdv** [comment]
Just confirming that this is not fixed by any work up to `0.12.6dev7` however, it is possible to pass a width to the widgetbox:

```
l = row(widgetbox(dropdown, data_table, width=350), p1)
```

**Comment by mattpap** [comment]
![image](https://user-images.githubusercontent.com/27475/53673895-8cfc2900-3c8a-11e9-95d1-925853df834e.png)
